gradle/libs.versions.toml

```diff
@@ -94,6 +99,14 @@ easyimage = { module = "com.github.jkwiecien:EasyImage", version.ref = "easyimag
 customactivityoncrash = { module = "cat.ereza:customactivityoncrash", version.ref = "customactivityoncrash" }
 zelory-compressor = { module = "id.zelory:compressor", version.ref = "zelory-compressor" }
 
+activity-compose = { module = "androidx.activity:activity-compose", version.ref = "activity-compose"}
+compose-bom = { module = "androidx.compose:compose-bom", version.ref = "compose-bom" }
+
+compose-ui = { module = "androidx.compose.ui:ui" }
```

Comment:
Is there a reason why you have didn't split up the module and name of a library? Not sure if it is bad practice to not split them, maybe we have to research this?

Example: 
`activity-compose = { module = "androidx.activity:activity-compose", version.ref = "activity-compose"}`
Could be written as:
`activity-compose = { module = "androidx.activity", name = "activity-compose", version.ref = "activity-compose"}`